ajax/libs/jquery.quicksearch/package.json

```diff
@@ -11,27 +11,30 @@
     },
     {
       "name": "Rik Lomas",
-      "url": "http://www.riklomas.co.uk/"
+      "url": "https://www.riklomas.co.uk/"
```

Comment:
Actually, I can't access this URL.

Maybe we don't have to change to `https`.
Because there is no SSL certification. I refer from https://www.ssllabs.com/ssltest/analyze.html?d=www.riklomas.co.uk

What do you think?
Thanks.

Reply:
Still not be modified yet.